src/server/main.py

```diff
@@ -57,5 +57,4 @@ def send_lib_file(path: str):
     app.logger.handlers = gunicorn_logger.handlers
     app.logger.setLevel(gunicorn_logger.level)
     sqlalchemy_logger = logging.getLogger("sqlalchemy")
-    sqlalchemy_logger.handlers = gunicorn_logger.handlers
-    sqlalchemy_logger.setLevel(gunicorn_logger.level)
+    sqlalchemy_logger.disabled = True
```

Comment:
The intent is to suppress `INFO` messages that simply print out the SQL query, because we already log these via structlog. However, I changed this line to simply set the log level to `ERROR` just so we don't miss out on any error messages.

Reply:
how about `WARN`, just in case?

Reply:
sure, fixed!